### PR TITLE
Add phone number information

### PR DESCRIPTION
**Pull request by paul-nameless**
(#148)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -121,7 +121,8 @@ Config file should be stored at `~/.config/tg/conf.py`. This is simple python fi
 ### Simple config:
 
 ```python
-PHONE = "[your phone number]"
+# should start with + (plus) and contain country code
+PHONE = "[phone number in international format]"
```
Comment (lightme16) [inline readme.md:125]: maybe let's add reexp to validate bad phone number?

**Comment by lightme16** [comment]
Also, let's create log file dir if it is missing as per https://github.com/paul-nameless/tg/issues/148#issuecomment-662258752

**Comment by paul-nameless** [comment]
Yes, I'll do it in separate patch

File: readme.md (hunk 1 above)
Comment (paul-nameless) [reply readme.md:125]: That's an option but I don't want to add unnecessary logic. Tdlib will tell if phone number is incorrect don't think we need to validate it as well

**Pull request merged**